### ISSUE TITLE
Application "disapears" after install and breaks Apache2

### ISSUE THREAD

**Issue by ErikMN** [oguzhaninan/Stacer#470]
After installing version 1.1.0-3 the application works fine on first run but after rebooting the PC it was nowhere to be found:

`➜ lnxuser@lnxuser-home  ~  locate stacer                              
/etc/apt/sources.list.d/oguzhaninan-ubuntu-stacer-focal.list
/etc/apt/trusted.gpg.d/oguzhaninan_ubuntu_stacer.gpg
/home/lnxuser/.config/stacer
/home/lnxuser/.config/stacer/settings.ini
/timeshift/snapshots/2021-11-27_16-29-47/localhost/var/lib/app-info/icons/ubuntu-focal-universe/48x48/stacer_stacer.png
/timeshift/snapshots/2021-11-27_16-29-47/localhost/var/lib/app-info/icons/ubuntu-focal-universe/64x64/stacer_stacer.png
/usr/share/applications/stacer.desktop
/usr/share/doc/stacer
/usr/share/doc/stacer/README.source
/usr/share/doc/stacer/changelog.Debian.gz
/usr/share/doc/stacer/copyright
/usr/share/icons/hicolor/128x128/apps/stacer.png
/usr/share/icons/hicolor/16x16/apps/stacer.png
/usr/share/icons/hicolor/256x256/apps/stacer.png
/usr/share/icons/hicolor/32x32/apps/stacer.png
/usr/share/icons/hicolor/64x64/apps/stacer.png
/var/lib/app-info/icons/ubuntu-focal-universe/48x48/stacer_stacer.png
/var/lib/app-info/icons/ubuntu-focal-universe/64x64/stacer_stacer.png
/var/lib/dpkg/info/stacer.list
/var/lib/dpkg/info/stacer.md5sums`

Apache2 would not start anymore. A reinstall of apache fixed the issue.

**Comment by alexmyczko** [comment]
what os/release is that (Ubuntu 20.04?)? how is stacer installed, from official apt repository or somehow else? what exact version?

**Comment by ErikMN** [comment]
Yes, Ubuntu 20.04 LTS, both tried to install via downloaded .deb and from PPA, version as described above.

After first install and reboot the executable had disappeared and could not be recovered by a reinstall.

**Comment by alexmyczko** [comment]
downloaded deb from where? using apt install stacer?

**Comment by ErikMN** [comment]
Yes using the APT repo.

**Comment by alexmyczko** [comment]
it is not in /usr/bin/stacer ?

**Comment by alexmyczko** [comment]
locate might need an updatedb

**Comment by ErikMN** [comment]
The executable was nowhere to be found, I uninstalled the application since it broke my Apache server.
